core/util/runPuppet.js

```diff
@@ -71,114 +74,116 @@ async function processScenarioView (scenario, variantOrScenarioLabelSafe, scenar
   if (chromeVersion < MIN_CHROME_VERSION) {
     console.warn(`***WARNING! CHROME VERSION ${MIN_CHROME_VERSION} OR GREATER IS REQUIRED. PLEASE UPDATE YOUR CHROME APP!***`);
   }
-
-  // --- BEFORE SCRIPT ---
-  var onBeforeScript = scenario.onBeforeScript || config.onBeforeScript;
-  if (onBeforeScript) {
-    var beforeScriptPath = path.resolve(engineScriptsPath, onBeforeScript);
-    if (fs.existsSync(beforeScriptPath)) {
-      await require(beforeScriptPath)(page, scenario, viewport, isReference, browser);
-    } else {
-      console.warn('WARNING: script not found: ' + beforeScriptPath);
+  
+  let result;
+  const puppetCommands = async () => {
+    // --- BEFORE SCRIPT ---
+    var onBeforeScript = scenario.onBeforeScript || config.onBeforeScript;
```

Comment:
Is there a reason for using `var`?

Reply:
the reason is that I copy pasted it from the chromy script 😜